apps/test/unit/code-studio/components/progress/UnitOverviewTopRowTest.jsx

```diff
@@ -239,4 +307,25 @@ describe('UnitOverviewTopRow', () => {
     );
     expect(wrapper.find(DropdownButton).length).toBe(0);
   });
+
+  it('renders student select dropdown if user is teacher and in teacher-local-nav-v2 experiment', () => {
+    jest
+      .spyOn(TeacherNavFlagUtils, 'showV2TeacherDashboard')
+      .mockImplementation(() => {
+        return true;
+      });
+
+    render(
+      <Provider store={store}>
+        <UnitOverviewTopRow
+          {...defaultProps}
+          publishedState="in_development"
+          scriptOverviewPdfUrl="/link/to/script_overview.pdf"
+          scriptResourcesPdfUrl="/link/to/script_resources.pdf"
+          viewAs={ViewType.Instructor}
+        />
+      </Provider>
+    );
+    screen.getByLabelText(i18n.viewingProgressFor());
```

Comment:
Should this also have an assertion that a specific student is selected or nothing is selected to start with?

Reply:
I'll add that. Should be set to "Me" by default